### PR TITLE
#165484337: Merge landing and welcome page design

### PR DESCRIPTION
**Pull request by olajide1234**
#### What does this PR do?
- Fixed landing page

#### Description of task to be completed?
- Improve header to welcome user upon signing in 
- Include search and filter components

#### How should this be manually tested?
- Sign in to the application and observe welcome greeting at the top of the page

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#165484337](https://www.pivotaltracker.com/story/show/165484337)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A

### CODE REVIEW

**Review by codinger41 — DISMISSED** [review]

**Review by youngestdj — DISMISSED** [review]

**Review by youngestdj — DISMISSED** [review]

**Review by codinger41 — DISMISSED** [review]

File: src/components/header/Index.jsx [hunk 1]
```diff
@@ -1,20 +1,26 @@
-/* eslint-disable no-unused-expressions */
 /* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/jsx-indent */
```
Comment (9jaswag) [inline src/components/header/Index.jsx]: @olajide1234 we're disabling indenting too?

File: src/components/header/Index.jsx [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+/* eslint-disable indent */
+/* eslint-disable no-unused-expressions */
```
Comment (9jaswag) [inline src/components/header/Index.jsx]: @olajide1234 why are we disabling this?

**Review by 9jaswag — CHANGES_REQUESTED** [review]

File: src/components/header/Index.jsx (hunk 1 above)
Comment (olajide1234) [reply src/components/header/Index.jsx]: This rule is now retained.

File: src/components/header/Index.jsx (hunk 2 above)
Comment (olajide1234) [reply src/components/header/Index.jsx]: Resolved. No rules are disabled in the page now.

**Review by codinger41 — DISMISSED** [review]

**Review by youngestdj — DISMISSED** [review]

**Review by codinger41 — APPROVED** [review]

**Review by youngestdj — APPROVED** [review]

**Review by 9jaswag — APPROVED** [review]

**Pull request merged**